src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -279,6 +279,18 @@ public void ComponentInitializationConfigurationError(string type, string error,
                 this.nameProvider.Name);
         }
 
+        [Event(
+            23,
+            Message = "ApplicationInsights configuration file '{0}' was not found.",
+            Level = EventLevel.Warning)]
+        public void ApplicationInsightsConfigNotFoundWarning(string directory, string appDomainName = "Incorrect")
```

Comment:
Resolved with next checkin. Thank you.

---
In reply to: [61342031](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/224#discussion_r61342031) [](ancestors = 61342031)